delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/ce/CatsScopeInitialization.scala

```diff
@@ -0,0 +1,66 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.ce
+
+import cats.effect.IO
+import ch.epfl.bluebrain.nexus.delta.kernel.effect.migration.toMonixBIOOps
+import ch.epfl.bluebrain.nexus.delta.sdk.ScopeInitialization
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.organizations.model.Organization
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.model.Project
+import monix.bio.{IO => BIO}
+
+/**
+  * Lifecycle hook for organization and project initialization. It's meant to be used for plugins to preconfigure an
+  * organization or project, like for example the creation of a default view or setting the appropriate permissions.
+  * Implementations should use a `many[ScopeInitialization]` binding such that all implementation are collected during
+  * the service bootstrapping.
+  */
+trait CatsScopeInitialization {
```

Comment:
I opted for the addition of this trait along with its the helper function as making a change to the ScopeInitialization would have had an impact across many plugins. We could slowly use this one as the migration goes on and then rename it.